doc/reference/dev/release.md

```diff
@@ -0,0 +1,63 @@
+# The Gerbil Release Process
+
+Congratulations, you are now a Gerbil maintainer,
+tasked with making a new release.
+Here is a checklist of all the steps you have to go through
+to make a successful release.
+
+## Get All the Changes Merged
+First, make sure your have all the features and bug fixes in that you need to be.
+Check the [PR queue](https://github.com/mighty-gerbils/gerbil/pulls) and the
+[Open Issues](https://github.com/mighty-gerbils/gerbil/issues) for any release blocker.
+
+## Regenerate the Bootstrap If Needed
+If there were any changes to the core language, [regenerate the bootstrap code](bootstrap).
+Include all the parts that have changed (runtime, prelude, expander, compiler).
+Make a dedicated PR just for this bootstrap upgrade.
+
+## Make Sure Gerbil Works on all Supported Targets
+Before you commit the above PR, build and test it at least on the following targets:
+Linux x86-64, macOS ARM64.
+[Ask for help on gitter](https://app.element.io/#/room/#gerbil-scheme_community:gitter.im)
+regarding platforms you can't support yourself.
+
+## Prepare a PR for the Release
+- Update the version everywhere in documentation
+- Update the version everywhere in in-repo release scripts (homebrew, guix)
+- Generate a `src/gerbil/runtime/version.ss` for the release
+- Temporarily remove `src/gerbil/runtime/version.ss` from the `.gitignore`
+- **TODO: tweak the build scripts so they work better without `git`?**
+- **TODO: also handle generation of Gambit version in `src/gambit/include/stamp.h` or some such?**
```

Comment:
we havent been doing this, and it may only be relevant foe the source tarball. I dont think we should do anything.

Reply:
Well, it is relevant for the source tarball, and maybe also for all stable releases on Nix, Guix, homebrew, etc.